static/docs/commands-reference/pull.md

```diff
@@ -35,28 +31,25 @@ Using the command `dvc status --remote REMOTE` determines whether the local
 cache has differences from the named remote cache.
```

Comment:
this is all misleading, if feels like we actually comparing local cache and some remote cache. It's only part of it. The important piece is that we actually looking for a list of data files references in the workspace that are missing in the local cache.

Reply:
Good clarification, I did not understand that until you've explained it here.